test/Microsoft.AspNet.Mvc.TagHelpers.Test/ScriptTagHelperTest.cs

```diff
@@ -580,17 +595,18 @@ public static TheoryData DoesNotRunWhenARequiredAttributeIsMissing_Data
             var globbingUrlBuilder = new Mock<GlobbingUrlBuilder>();
             globbingUrlBuilder.Setup(g => g.BuildUrlList("/js/site.js", "**/*.js", null))
                 .Returns(new[] { "/js/site.js", "/common.js" });
-            var helper = new ScriptTagHelper
+
+            var helper = new ScriptTagHelper(
+                logger.Object,
+                hostingEnvironment,
+                MakeCache(),
+                new CommonTestEncoder(),
+                new CommonTestEncoder())
             {
                 GlobbingUrlBuilder = globbingUrlBuilder.Object,
-                Logger = logger.Object,
-                HostingEnvironment = hostingEnvironment,
                 ViewContext = viewContext,
                 Src = "/js/site.js",
                 SrcInclude = "**/*.js",
-                HtmlEncoder = new CommonTestEncoder(),
-                JavaScriptEncoder = new TestJavaScriptEncoder(),
```

Comment:
Why? there's 1-2 tests in all of this code that use our own version of something that's been commonized. This is an improvement, and it was faster to do this and try and be pedantic about what the tests did before.